metroae

```diff
@@ -1285,7 +1285,7 @@ function docker_exec_interactive {
         environment=" -e METROAE_PASSWORD=$METROAE_PASSWORD"
     fi
     if [[ ! -z $SSHPASS_PASSWORD ]]; then
-        sshpass="sshpass -p$SSHPASS_PASSWORD"
+        sshpass="sshpass -p \"'$SSHPASS_PASSWORD'"
```

Comment:
What is the escaped double quote for?

Reply:
it is for escaping sequence for escaping single-quotes.

Reply:
I don't understand.  To me this looks like:

sshpass -p "'$SSHPASS_PASSWORD'

isn't there an extra double quote?